CHANGELOG.md

```diff
@@ -44,7 +44,12 @@ Ref: https://keepachangelog.com/en/1.0.0/
 ### State Machine Breaking Changes
 
 ### Features
+* (toolsets/oracle-operator) [\#2](https://github.com/certikfoundation/shentu/pull/2) Added toolset oracle-operator.
+* (toolsets/oracle-operator) [\#5](https://github.com/certikfoundation/shentu/pull/5) Added multi-client support.
 
 ### Improvements
+* (x/oracle) [\#6](https://github.com/certikfoundation/shentu/pull/6) Updated events and added useful fields in task types.
```

Comment:
Seems like this is API breaking

Reply:
The newly added fields can be automatically generated. Is the change of event also a kind of API breaking?

Reply:
Updated in API Breaking Changes.